perf/perf_script_bug.py

```diff
@@ -48,14 +48,18 @@ def setUp(self):
                         os.path.join(self.teststmpdir, 'Makefile'))
         build.make(self.teststmpdir)
         os.chdir(self.teststmpdir)
+        if self.distro_name == 'rhel':
+            self.param = 3
```

Comment:
can we controlled this value from YAML so in that case, we need not check distro version as well and we can have explained these  value in  yaml itself

Reply:
I would suggest using something like https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/generic/service_check.py.data/services.cfg as it is not something which changes but needs to be maintained separately case-by-case. It can also have specific distro versions.

Reply:
yup agree just my intention here is we need not touch code for some value change which depends on distro or dynamic

Reply:
@shirishaganta Can you adopt this change here?

Reply:
@harish-24 @PraveenPenguin I have adopted the changes here which you people are suggested. Please have a look into it